torch/_tensor_str.py

```diff
@@ -248,8 +248,7 @@ def _str(self):
         if self.device.type == 'cpu' or torch.cuda.current_device() != self.device.index:
             suffixes.append('device=\'' + str(self.device) + '\'')
 
-    has_default_dtype = self.dtype == torch.get_default_dtype() or self.dtype == torch.int64
-
+    has_default_dtype = self.dtype == torch.get_default_dtype() or self.dtype == torch.int64 or self.dtype == torch.bool
```

Comment:
Do you think it's better to replace these consecutive `==`s with

    self.dtype in (torch.get_default_dtype(), torch.int64, torch.bool)